backend/tests/Feature/CreateUserEndpointTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Tests\Feature;
+
+use Nuwave\Lighthouse\Testing\MakesGraphQLRequests;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Tests\TestCase;
+
+class CreateUserEndpointTest extends TestCase
+{
+    use MakesGraphQLRequests;
+    use RefreshDatabase;
+    /**
+     * A basic feature test example.
+     *
+     * @return void
+     */
+    public function testEmptyName()
+    {
+        $response = $this->graphQL('
+        
+            mutation {
+                createUser(user: {name: "", email: "brandnew@gmail.com", password: "KajSu8viptUrz&", username: "testusername"}) {
+                    username
+                }
+            }
+        ');
+
+        $response->assertJsonPath("data.createUser.username", "testusername");
+
+
+    }
+
+    public function testMissingName()
+    {
+        $response = $this->graphQL('
+        
+            mutation {
+                createUser(user: {email: "brandnew@gmail.com", password: "KajSu8viptUrz&", username: "testusername"}) {
+                    username
+                }
+            }
+        ');
+
+        $response->assertJsonPath("data.createUser.username", "testusername");
+
```

Comment:
Might be useful to also assert that `data.createUser.name` is null. I'll see if I can come up with something.

Reply:
I added an assertion to check that the value of this property is null. Let me know what you think.